Makefile

```diff
@@ -306,8 +306,8 @@ utils/config.ml: utils/config.mlp config/Makefile
 	    -e 's|%%ASM%%|$(ASM)|' \
 	    -e 's|%%ASM_CFI_SUPPORTED%%|$(ASM_CFI_SUPPORTED)|' \
 	    -e 's|%%BYTECCLIBS%%|$(BYTECCLIBS)|' \
-	    -e 's|%%BYTECODE_C_COMPILER%%|$(BYTECODE_C_COMPILER)|' \
```

Comment:
Do we know if OCaml projects use variables such as this one? If possible, a global grep would be useful. Debian's codesearch gave me no match but I might be using it incorrectly.